dist/yaml/ovnkube.yaml

```diff
@@ -0,0 +1,176 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: ovnkube
+  namespace: openshift-sdn
+  annotations:
+    kubernetes.io/description: |
+      This daemon set launches the OpenShift networking components (kube-proxy, DNS, and ovn-k8s-cni-overlay).
+      It expects that OVS is running on the node.
+    image.openshift.io/triggers: |
```

Comment:
Same as ovnkube-master.yaml, let's remove this annotation.

Reply:
OK